src/stdlib_stats_distribution_uniform.fypp

```diff
@@ -0,0 +1,462 @@
+#:include "common.fypp"
+#:set RC_KINDS_TYPES = REAL_KINDS_TYPES + CMPLX_KINDS_TYPES
```

Comment:
Additional (stupid) question:
in [this comment](https://github.com/fortran-lang/stdlib/pull/272/files#r680254976), it is mentioned that the method cannot be extrapolated to 128 bit. Therefore, does it support well the type `qp`?

Reply:
It is unclear whether this method supports 128bit type. Someone has to carry out similar research on 128 bit float number using 128 bit integer number generator.